docs/en/operations/table_engines/mergetree.md

```diff
@@ -161,10 +164,33 @@ The number of columns in the primary key is not explicitly limited. Depending on
 
 - Provide additional logic when data parts merging in the [CollapsingMergeTree](collapsingmergetree.md#table_engine-collapsingmergetree) and [SummingMergeTree](summingmergetree.md#table_engine-summingmergetree) engines.
 
-    You may need many fields in the primary key even if they are not necessary for the previous steps.
+    If you need this, it makes sense to specify the *sorting key* that is distinct from the primary key.
```

Comment:
If you need this -> In some cases ?

Reply:
This comment applies specifically to the case when the user needs additional merge logic (specified in the bullet point above).